src/System.CommandLine/Properties/xlf/Resources.ru.xlf

```diff
@@ -1,216 +1,216 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en" target-language="ru" original="../Resources.resx">
     <body>
       <trans-unit id="ArgumentConversionCannotParse">
         <source>Cannot parse argument '{0}' as expected type {1}.</source>
-        <target state="new">Cannot parse argument '{0}' as expected type {1}.</target>
+        <target state="new">Невозможно разобрать аргумент '{0}' ожидается тип {1}.</target>
         <note />
       </trans-unit>
       <trans-unit id="ArgumentConversionCannotParseForCommand">
         <source>Cannot parse argument '{0}' for command '{1}' as expected type {2}.</source>
-        <target state="new">Cannot parse argument '{0}' for command '{1}' as expected type {2}.</target>
+        <target state="new">Невозможно разобрать аргумент '{0}' для команды '{1}' ожидается тип {2}.</target>
         <note />
       </trans-unit>
       <trans-unit id="ArgumentConversionCannotParseForOption">
         <source>Cannot parse argument '{0}' for option '{1}' as expected type {2}.</source>
-        <target state="new">Cannot parse argument '{0}' for option '{1}' as expected type {2}.</target>
+        <target state="new">Невозможно разобрать аргумент '{0}' для опции '{1}' ожидается тип {2}.</target>
         <note />
       </trans-unit>
       <trans-unit id="CommandExpectsFewerArguments">
         <source>Command '{0}' expects no more than {1} arguments, but {2} were provided.</source>
-        <target state="new">Command '{0}' expects no more than {1} arguments, but {2} were provided.</target>
+        <target state="new">Команда '{0}' ожидает не более {1} аргументов, но {2} предоставлено.</target>
         <note />
       </trans-unit>
       <trans-unit id="CommandExpectsOneArgument">
         <source>Command '{0}' expects a single argument but {1} were provided.</source>
-        <target state="new">Command '{0}' expects a single argument but {1} were provided.</target>
+        <target state="new">Команда '{0}' ожидает единственный аргумент но {1} предоставлено.</target>
         <note />
       </trans-unit>
       <trans-unit id="CommandNoArgumentProvided">
         <source>No argument was provided for Command '{0}'.</source>
-        <target state="new">No argument was provided for Command '{0}'.</target>
+        <target state="new">Нет аргументов для команды '{0}'.</target>
```

Comment:
nit: according to google translate it means "No argument for command". We are missing the "was provided"